src/java/org/apache/cassandra/cql3/statements/schema/CreateIndexStatement.java

```diff
@@ -229,8 +237,9 @@ private void validateIndexTarget(TableMetadata table, IndexMetadata.Kind kind, I
 
         AbstractType<?> baseType = column.type.unwrap();
 
-        if ((kind == IndexMetadata.Kind.CUSTOM) && !SchemaConstants.isValidName(target.column.toString()))
-            throw ire(INVALID_CUSTOM_INDEX_TARGET, target.column, SchemaConstants.NAME_LENGTH);
+        // This check will be removed with CASSANDRA-20235
```

Comment:
remove this line

Reply:
This line is put on purpose as such check must not exist. It indicates that fixing it is left to a follow up work. Can you explain your reasoning for your comment?

Reply:
The codebase is not a notepad. You already have Jira, so why do you need this kind of meaningless record? Have you seen similar notepad messages in recent Cassandra code?

Reply:
cc @netudima @smiklosovic

Reply:
`Can you explain your intention with this phrase, so I am not misinterpreting it?`
When I saw this comment, I felt like the author had written a memo to himself, asking him to delete it in which patch.
1、it doesn’t even have a basic TODO
2、I think event you mark this down, you should have a basic test case to cover it . what if the author forget to remove it in CASSANDRA-20235 ? and how we can find it ? Of course, if we are lucky, there are other test cases that can cover it.

Reply:
> what if the author forget to remove it in [CASSANDRA-20235](https://issues.apache.org/jira/browse/CASSANDRA-20235) ?

This is one of the reason for the comment. I don't have a better idea how to help to eventually remove the check.